README.md

```diff
@@ -210,6 +210,12 @@ Haskell's [groupBy](http://zvon.org/other/haskell/Outputlist/groupBy_f.html).
 
 **Note:** Intersperse can be used with JSX components! Specially with the `differentLast` iterator:
 
+### replaceElementBy
+`(fn(array_element), value, array) -> array` Replaces an element in an array with `value` based on the boolean result of a function `fn`.
+
+### replaceElement
+`(a, b, array) -> array` Replaces all elements equal to `a` in an array with `b`.
```

Comment:
for the docs, let's give `a` and `b` clearer names

Reply:
Done